README.md

```diff
@@ -88,6 +89,7 @@ For the purpose of setup, your project's root folder (where you can find your `.
 #### 1. Download Binaries
 
 * Download the `AmazonChimeSDK` and `AmazonChimeSDKMedia` binaries from the latest [release](https://github.com/aws/amazon-chime-sdk-ios/releases/latest).
+  * If you like to use background blur and background replacement, also download the `AmazonChimeSDKMachineLearning` binary. Otherwise, you can ignore all references to `AmazonChimeSDKMachineLearning` in the instructions below.
```

Comment:
Is there any plan to publish the ML SDK to CocoaPods?

Reply:
Yes